pkg/postage/postagecontract/contract.go

```diff
@@ -123,6 +129,38 @@ func (c *postageContract) sendCreateBatchTransaction(ctx context.Context, owner
 	return receipt, nil
 }
 
+func (c *postageContract) sendTopUpBatchTransaction(ctx context.Context, batchID []byte, topUpAmount *big.Int) (*types.Receipt, error) {
+
+	callData, err := postageStampABI.Pack("topUp", common.BytesToHash(batchID), topUpAmount)
+	if err != nil {
+		return nil, err
+	}
+
+	request := &transaction.TxRequest{
+		To:       &c.postageContractAddress,
+		Data:     callData,
+		GasPrice: sctx.GetGasPrice(ctx),
+		GasLimit: 160000,
+		Value:    big.NewInt(0),
```

Comment:
please set the `Description` field. ideally also add it to the create batch and approve request.